commit 638f95c3b0cffb237eb2a69fadb71c565c3dd014
Author: Samuli Suominen <ssuominen@gentoo.org>
Date:   Fri Feb 18 14:00:32 2011 -0300

    Fix notify plugin build with libnotify 0.7.
    
    Patch by Samuli Suominen.
    See: https://bugs.gentoo.org/show_bug.cgi?id=353848
    
    Sent-by: Alexis Ballier <aballier@gentoo.org>
    Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>

Index: vlc-1.1.7/modules/misc/notify/notify.c
===================================================================
--- vlc-1.1.7.orig/modules/misc/notify/notify.c
+++ vlc-1.1.7/modules/misc/notify/notify.c
@@ -38,6 +38,10 @@
 #include <gdk-pixbuf/gdk-pixbuf.h>
 #include <libnotify/notify.h>
 
+#ifndef NOTIFY_CHECK_VERSION
+#define NOTIFY_CHECK_VERSION(x,y,z) 0
+#endif
+
 /*****************************************************************************
  * Module descriptor
  ****************************************************************************/
@@ -312,7 +316,12 @@ static int Notify( vlc_object_t *p_this,
     }
 
     notification = notify_notification_new( _("Now Playing"),
-                                            psz_temp, NULL, NULL );
+                                            psz_temp, NULL
+#if NOTIFY_CHECK_VERSION (0, 7, 0)
+					    );
+#else
+					    , NULL );
+#endif
     notify_notification_set_timeout( notification,
                                 var_InheritInteger(p_this, "notify-timeout") );
     notify_notification_set_urgency( notification, NOTIFY_URGENCY_LOW );
